internal/internal_test.go

```diff
@@ -1114,6 +1115,23 @@ func TestGetResponseHeaders(t *testing.T) {
 	if err == nil {
 		t.Fatal("Expected error but got nil")
 	}
+
+	testHeaders = make(map[string]interface{})
+	testHeaders["foo"] = "bar"
+	testHeaders2 := make(map[string]interface{})
+	testHeaders2["foo"] = "baz"
+	testHeadersSlices := make([]interface{}, 0)
+	testHeadersSlices = append(testHeadersSlices, testHeaders)
+	testHeadersSlices = append(testHeadersSlices, testHeaders2)
+	input["headers"] = testHeadersSlices
+
+	result, err = getResponseHeaders(input)
+	if err != nil {
+		t.Fatalf("Expected no error but got %v", err)
+	}
+	if len(result) != 2 {
+		t.Fatalf("Expected two header but got %v", len(result))
+	}
```

Comment:
Would you think it be useful to have more concrete assertions on what is returned there? 🤔

Reply:
🤔 checking duplicated headers ?

Reply:
Well I guess we've got it covered by asserting that there's two headers. What else should they be than `bar` and `baz`? 🤔 

It's up to you, not blocking this from getting merged, but we could do something like (⚠️ untested sketch)
```go  
seen := map[string]int{}
for _, hdr := range result {
    seen[hdr.Header.Value]++
}
if seen["bar"] != 1 || seen["baz"] != 1 {
    t.Errorf("expected 'bar' and 'baz', got %v", seen)
}
```